README.md

```diff
@@ -331,16 +331,16 @@ We do not use `input[type=date]`, but instead combine three `input[type=text]` w
 ```
 [_See the full markup for date inputs in the registry_](https://registry.origami.ft.com/components/o-forms#date)
 
-#### Anchors
-Anchors are an entirely separate entity. They _look_ like a form element but do not function as one.
+#### Pseudo Radio Links
+Anchor elements are an entirely separate entity. They _look_ like a form element but do not function as one. They are styled to look like a box-like radio button.
 
 ```html
-<div class="o-forms-input o-forms-input--pseudo-radio-box ">
-		<a class="o-forms__anchor o-forms__anchor--current" href="#">Anchor Link A</a>
-		<a class="o-forms__anchor" href="#">Anchor Link B</a>
+<div class="o-forms-input o-forms-input--pseudo-radio-link">
+		<a class="o-forms-input__link--current" href="#">Link A</a>
```

Comment:
Should we include the element and modifier class, even if the element class doesn't currently have styles? `o-forms-input__link o-forms-input__link--current`